CRM/Contact/BAO/Relationship.php

```diff
@@ -551,12 +551,13 @@ public static function getContactRelationshipType(
       }
     }
 
-    $contactSubType = array();
     if ($contactId) {
       $contactType = CRM_Contact_BAO_Contact::getContactType($contactId);
       $contactSubType = CRM_Contact_BAO_Contact::getContactSubType($contactId);
     }
 
+    $contactSubType = (array) $contactSubType;
```

Comment:
No, as $contactSubType defaults to NULL, an NULL can be casted to an empty array, see https://3v4l.org/nrnGP